charts/rancher-cis-benchmark/rancher-cis-benchmark/2.0.1/Chart.yaml

```diff
@@ -0,0 +1,19 @@
+annotations:
+  catalog.cattle.io/auto-install: rancher-cis-benchmark-crd=match
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/display-name: CIS Benchmark
+  catalog.cattle.io/namespace: cis-operator-system
+  catalog.cattle.io/os: linux
+  catalog.cattle.io/provides-gvr: cis.cattle.io.clusterscans/v1
+  catalog.cattle.io/release-name: rancher-cis-benchmark
+  catalog.cattle.io/type: cluster-tool
+  catalog.cattle.io/ui-component: rancher-cis-benchmark
+apiVersion: v1
+appVersion: v1.0.5
```

Comment:
This needs to be `v1.0.6` to match the tag of `rancher/cis-operator`

It can be edited in `packages/rancher-cis-benchmark/charts/Chart.yaml`

Reply:
Thanks, done. Will I have to do this as well for [my system-charts PR](https://github.com/rancher/system-charts/pull/534)?

Reply:
yes, you need to update the `Chart.yaml` there as well.